### PR TITLE
[opensource] Apply Apache 2.0 with LICENSE and NOTICE

### PR DESCRIPTION
**Pull request by Martouta**
As @unleashed said in https://github.com/3scale/porta/pull/372#issuecomment-443411060

> APPLYING THE LICENSE TO NEW SOFTWARE
> To apply the ALv2 to a new software distribution, include one copy of the license text by copying the file:
> 
> http://www.apache.org/licenses/LICENSE-2.0.txt
> 
> into a file called LICENSE in the top directory of your distribution. If the distribution is a jar or tar file, try to add the LICENSE file first in order to place it at the top of the archive. This covers the collective licensing for the distribution.
> 
> In addition, a correct NOTICE file MUST be included in the same directory as the LICENSE file.
> 
> Each original source document (code and documentation, but excluding the LICENSE and NOTICE files) SHOULD include a short license header at the top. If the distribution contains documents not covered by CLA, CCLA or Software Grant (such as third-party libraries) then see the policy guide.
> 
> UPDATING EXISTING SOFTWARE
> In brief, the aim is to achieve a final distribution as described above in applying the license to new software. Some conversion tools are listed here.

### CODE REVIEW

File: NOTICE [hunk 1]
```diff
@@ -0,0 +1,15 @@
+   Red Hat 3scale API Management Platform Porta
```
Comment (Martouta) [inline NOTICE]: I didn't know if I should write `Porta` or `System` 🤷‍♀️

Comment (unleashed) [reply NOTICE]: I am not even sure we say "Platform" anymore

Comment (hallelujah) [reply NOTICE]: I would just avoid using Platform just to not being confused.

Comment (Martouta) [reply NOTICE]: ok, so should I push with `Red Hat 3scale API Management User Interface` instead? @hallelujah 
It is what we currently have in the description of the repository:
image

Comment (hallelujah) [reply NOTICE]: I really do not know. But yes. Something along:

"Red Hat 3scale API Management console, Porta"
@thomasmaas

**Comment by thomasmaas** [comment]
Admin console is the term we use in docs when we talk about the actual user interface of the admin portal. 

But porta is more than that. There’s an API, there’s billing, developer portal.

**Pull request merged**